pyproject.toml

```diff
@@ -14,7 +14,11 @@ classifiers = [
   "Programming Language :: Python :: 3",
   "Operating System :: OS Independent",
 ]
-dependencies = []
+dependencies = [
+  "requests==2.*",
+  "geojson==3.*",
+  "requests_oauthlib==1.*",
```

Comment:
We typically pin our dependencies to specific versions, rather than major versions

Reply:
The issue is that if we pin to specific versions here, and users also install a different library that uses requests, that is pinned to a different version, then there will be a conflict. The docs recommend against it:

> It is not considered best practice to use install_requires to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/

We don't rely on any particularly advanced features of any of these libraries so we should be as liberal as possible in what we accept until there is a demonstrated issue with a particular version.